docs/Errors.md

```diff
@@ -7,22 +7,22 @@
 ### Error Handling In Node.js
 
 #### Uncaught Errors
-In Node.js, uncaught errors are likely to cause memory leaks, file descriptor leaks and other major production issues. [Domains](https://nodejs.org/en/docs/guides/domain-postmortem/) were introduced to try fix this issue, but did not. 
+In Node.js, uncaught errors are likely to cause memory leaks, file descriptor leaks and other major production issues. [Domains](https://nodejs.org/en/docs/guides/domain-postmortem/) were introduced to try to fix this issue, but did not. 
 
 Given that it is not possible to process all uncaught errors sensibly, the best way to deal with them is to [crash](https://nodejs.org/api/process.html#process_warning_using_uncaughtexception_correctly). 
 
-#### Catching Errors In Promises
+#### Catching Errors in Promises
```

Comment:
See my comment https://github.com/fastify/fastify/pull/2871#discussion_r581007454

Reply:
Removed title style change commit in https://github.com/fastify/fastify/commit/858911aeb6e8ff37410300ddc01e764e75f8b96b